.storybook/experimental.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, OnInit, AfterViewInit } from "@angular/core";
+import { ExperimentalService } from "../src/experimental.module";
+
+@Component({
+	selector: "app-experimental-component",
+	template: ``
+})
+export class ExperimentalComponenent implements AfterViewInit {
+	constructor(protected experimental: ExperimentalService) {
+		experimental.isExperimental = false;
```

Comment:
I can't find any place in the PR where experimental is set to true, what am I missing?

Reply:
We could drop this, the main benefit is it gives us a single place to handle storybooks experimental state. For actual development we need to toggle the `preview.scss` flag and this flag.

Reply:
The functionality part and templates still change based on `ExperimentalService`'s `isExperimental`, no?

Reply:
> templates still change based on ExperimentalService's isExperimental

Let me poke it again to make sure, but it wasn't updating dynamically.

Reply:
🤦‍♂️ yeah I totally messed up the first time I tried implementing ... works way better now